spec/ios_extract_keys_from_strings_files_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'tmpdir'
+
+describe Fastlane::Actions::IosExtractKeysFromStringsFilesAction do
+  let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_extract_keys_from_strings_files') }
+
+  it 'extracts the right keys from Localizable.strings to InfoPlist.strings for all locales but the base one' do
+    Dir.mktmpdir('a8c-release-toolkit-tests-') do |tmp_dir|
```

Comment:
FYI, we also have the following helper that saves some typing:

https://github.com/wordpress-mobile/release-toolkit/blob/06d9f926f2983d7c0b00b414746c83eff5b5b5f4/spec/spec_helper.rb#L70-L77

I'm fine using either one or the other. Calling `Dir.mktmpdir` removes one abstraction layer, but I find the helper makes the code less noisy. 🤷 ⚖️

Reply:
Ohhh I always forgot about this one, good point 👍 

I might do a separate PR to update all the `Dir.mktmpdir` calls in other specs as well to update them all to `in_tmp_dir` for consistency 🤔

Reply:
Addressed at least for this new action in 47d1e39816e849ed59fa089ae3b5ac0b22cf5576